sys/arm64/arm64/exception.S

```diff
@@ -26,6 +26,7 @@
  */
 
 #include <sys/elf_common.h>
+#include <sys/intr.h>
```

Comment:
Identical to above.  The only place which should #include <sys/intr.h> are the machine/intr.h headers. Everywhere else should pick up sys/intr.h by #include <machine/intr.h>.